android-components/components/browser/engine-gecko/src/test/java/mozilla/components/browser/engine/gecko/GeckoEngineSessionTest.kt

```diff
@@ -294,6 +294,7 @@ class GeckoEngineSessionTest {
         var observedCanGoBack = false
         var observedCanGoForward = false
         var cookieBanner = CookieBannerHandlingStatus.HANDLED
+        var displaysProduct = false
```

Comment:
Would this be better left as isProductUrl instead of making another variable name

Reply:
Now that I see the automation failure I remembered that the reason for the rename was that otherwise it confuses this variable with the parameter name from the function. This is just a test, it's ok to have a different name. Reverting this